### ISSUE TITLE
Libraries not getting added properly

### ISSUE THREAD

**Issue by shivams2u2** [kriswiner/MPU9250#277]
Hello sir, 
I am a beginner and i have bought Arduino mega and arduino MPU-9250. I am trying to find roll, pitch and yaw without display but i am not able to add libraries properly i guess. 
I have copied these libraries which are being uploaded by you in arduino library folder. 

I have attached  the screenshot of the code and error.

![capture](https://user-images.githubusercontent.com/39727383/40649443-90bbca70-634e-11e8-92b8-5380b0a20e33.PNG)
![capture1](https://user-images.githubusercontent.com/39727383/40649444-90e9e57c-634e-11e8-8472-8e5fb1b58c1b.PNG)

**Comment by kriswiner** [comment]
Why not just run the sketch as is?

You've added libraries here that are not part of the original sketch, I
can't help you with that...

**Comment by ssammut95** [comment]
I don't think this has to do with libraries. Use the code Kris called "MPU9250BasicAHRS.ino" and remove all instances of the Adafruit screen if you are not using it.

**Comment by ssamm95** [comment]
I don't think this has to do with libraries. Use the code Kris called "MPU9250BasicAHRS.ino" and remove all instances of the Adafruit screen if you are not using it.
